src/identity/allowances/actions/creators/index.ts

```diff
@@ -0,0 +1,24 @@
+// Types
+import {RemoteDataState} from 'src/types'
+
+export const SET_ORG_CREATION_ALLOWANCES = 'SET_ORG_CREATION_ALLOWANCES'
+export const SET_ORG_CREATION_ALLOWANCES_STATUS =
+  'SET_ORG_CREATION_ALLOWANCES_STATUS'
+
+export type Actions =
+  | ReturnType<typeof setOrgCreationAllowances>
+  | ReturnType<typeof setOrgCreationAllowancesStatus>
+
+export const setOrgCreationAllowances = allowances =>
```

Comment:
Can we add typing for the allowance here?  Should be able to import from unityRoutes or the UI API layer.